### PR TITLE
Store current page number and page size of react-table in redux store

### LINKED ISSUE #55: Store the pagination values in redux store (1)
Store the pagination values like page number and no of rows per page in redux store.

### PR DESCRIPTION
**Pull request by amit-mb**
This PR stores the current  page number and page-size of react-table with tabId as the key in redux store so that the page numbers and size do not reset to default values when we change tabs or close the popup.

Fixes #55

### CODE REVIEW

File: app/modules/recordings.ts [hunk 1]
```diff
@@ -1,34 +1,76 @@
...
+        ...state,
+        PageDetails: {
+          ...state.PageDetails,
+          [action.payload.tabId]: {
+            ...state.PageDetails[action.payload.tabId],
+            currentPageNumber: action.payload.currentPageNumber
+          }
+        }
+      };
+    case actionType.ROW_SIZE_CHANGE:
```
Comment (revathskumar) [inline app/modules/recordings.ts]: As discussed, These two can be clubbed into single action.

File: app/popup.tsx [hunk 2]
```diff
@@ -127,6 +141,10 @@ export class Popup extends React.Component<POPUP_PROPS & DispatchProps, {}> {
           statusCodes={this.props.statusCodes}
           handleContentTypeChange={this.props.handleContentTypeChange}
           contentType={this.props.contentType}
+          handlePageNumberChange = {this.props.handlePageNumberChange}
+          handleRowSizeChange = {this.props.handleRowSizeChange}
+          PageDetails = {this.props.PageDetails}
```
Comment (revathskumar) [inline app/popup.tsx]: Need to send only the `pageDetails` of the currentTab.

File: app/request_list.tsx [hunk 3]
```diff
@@ -85,11 +94,23 @@ const RequestList = (props: RequestObj) => {
       showPagination={true}
       showPaginationTop={false}
       showPaginationBottom={true}
-      pageSize={10}
-      freezeWhenExpanded
+      defaultPageSize={10}
+      page={props.PageDetails[props.tabId] ? props.PageDetails[props.tabId].currentPageNumber : 0}
```
Comment (revathskumar) [inline app/request_list.tsx]: should the default pageNumber 0 or 1?

Comment (amit-mb) [reply app/request_list.tsx]: The page numbers start with `0`. It uses `index`. However, the pageNumber is displayed as `1` on view.

**Pull request merged**